apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/ConsumerConstructorInterceptor.java

```diff
@@ -22,18 +22,49 @@
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 /**
- *
+ * @author zhang xin, stalary
```

Comment:
Please remove this, we don't recommend the author information in the ASF.

Reply:
yep checkstyle maven plugin was failing. Will remove this in final pull request.

Reply:
Addressed in updated PR